engine-rest/engine-rest-openapi/src/main/templates/paths/variable-instance/get.ftl

```diff
@@ -0,0 +1,111 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/variable-instance/get-query/index.html -->
+{
+  <@lib.endpointInfo
+      id = "getVariableInstances"
+      tag = "Variable Instance"
+      summary = "Get Variable Instances"
+      desc = "Query for variable instances that fulfill given parameters. Parameters may be the
+              properties of variable instances, such as the name or type. The size
+              of the result set can be retrieved by using the [Get Variable Instance
+              Count](${docsUrl}/reference/rest/variable-instance/get-query-count/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign requestMethod="GET"/>
+    <#include "/lib/commons/variable-instance-query-params.ftl" >
+    <#assign last = false >
+
+    <@lib.parameters
+        object = params
+    />
+
+    <#include "/lib/commons/sort-params.ftl" >
+
+    <#include "/lib/commons/pagination-params.ftl" >
+    <#assign last = true >
+
+    <#include "/lib/commons/deserialize-values-parameter.ftl">
+
+],
+
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        dto = "VariableInstanceDto"
+        array = true
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "description": "GET `/variable-instance?processInstanceIdIn=aProcessInstanceId,anotherProcessInstanceId&variableValues=amount_gteq_5,amount_lteq_200`",
+                       "value": [
+                         {
+                           "id": "someId",
+                           "name": "amount",
+                           "type": "Integer",
+                           "variableType": "integer",
+                           "value": 5,
+                           "processDefinitionId": "aProcessDefinitionId",
+                           "processInstanceId": "aProcessInstanceId",
+                           "executionId": "b68b71c9-e310-11e2-beb0-f0def1557726",
+                           "taskId": null,
+                           "batchId": null,
+                           "activityInstanceId": "Task_1:b68b71ca-e310-11e2-beb0-f0def1557726",
+                           "caseExecutionId": null,
+                           "caseInstanceId": null,
+                           "serializationConfig": null,
```

Comment:
```suggestion
                           "serializationConfig": null,
                           "errorMessage": null,
```

Reply:
It appears that variableType and serializationConfig are also incorrect and not actually properties of this dto